bears/scss/SCSSLintBear.py

```diff
@@ -1,5 +1,19 @@
+import yaml
+
 from coalib.bearlib.abstractions.Linter import linter
 from coalib.bears.requirements.GemRequirement import GemRequirement
+from coalib.bears.requirements.PipRequirement import PipRequirement
+
+
+def convert_setting(setting):  # pragma: no cover
```

Comment:
there's no reason for not covering this function it's perfectly testable. Also the name is a bit generic, the conversion is specific to scsslint